test/v2/unit/recovery.ts

```diff
@@ -455,7 +454,7 @@ describe('Recovery:', function() {
         ignoreAddressTypes: ['p2wsh', 'p2shP2wsh']
       });
 
-      recovery.transactionHex.should.equal('0100000001edda677a92037140f01cacc1ac0e22391adb9bf4dbce1c4c218822128ac6fd5301000000b500483045022100c70ade07a526cdfdf29e0c16bd02ef38005695e2b140b937dbe2578253bee90402206611a571593272630a05406d653d5dc6950152665f24a0ac19fb006c2db33f8c014c69522103944ef399da9b45f26407dc3d7abd3544ef6fe6c32236bd6fe39f0db339f267a1210281941b32001f6fd10140e7a8f9a8df881f28d96f172528143e275d96c951a0422103a8ad848ac08c1fc598a91cb0786d6b663084ed3e399072de7c15c397629f2c6f53aeffffffff0288f400000000000017a91423dd90d164d5dfd07ba41814142a5bff0ad5446f87e00f97000000000017a91405437769516c39063ef0c4ed50cd323495cc913d8700000000');
+      recovery.transactionHex.should.equal('0100000001edda677a92037140f01cacc1ac0e22391adb9bf4dbce1c4c218822128ac6fd5301000000b4004730440220166d99f7b75b5d7aa470a675f0cb16ae27d48afc97312856923c4fd8c951422002201d62c537b3da7a787b0ef23f9075004df8db19376b521f9828cf5ec24bd9b741014c69522103944ef399da9b45f26407dc3d7abd3544ef6fe6c32236bd6fe39f0db339f267a1210281941b32001f6fd10140e7a8f9a8df881f28d96f172528143e275d96c951a0422103a8ad848ac08c1fc598a91cb0786d6b663084ed3e399072de7c15c397629f2c6f53aeffffffff02b8ec00000000000017a91423dd90d164d5dfd07ba41814142a5bff0ad5446f87e00f97000000000017a91405437769516c39063ef0c4ed50cd323495cc913d8700000000');
```

Comment:
same here, couldn't find the inputs

Reply:
looking for them now

Reply:
@dacruz21 wrote the original nockDashRecovery test vectors, so maybe he knows. it doesn't seem that these test vectors are based on real transactions on testnet or mainnet.
its ok by me, lets just make sure to do a full dash recovery before releasing

Reply:
That's strange. Yeah, I generated all of these fixtures by trying to recover real unspents on testnet, adding the txhex to the fixture, then broadcasting the txhex to make sure it was valid. So none of these txhexs should be spendable anymore but they should be valid.

Reply:
@dannybitgo What do you mean by saying you couldn't find the inputs? As in you couldn't find the inputs altogether (spent or unspent)? Or you couldn't find them as unspents? If it is the latter then this is expected as @dacruz21 said he already broadcasted these txes. If it is the former I think we should chat irl.

Reply:
when i ran that verifySigs script that I sent you @paddyquinn, it failed because the input txids on the transaction were not found by the block explorer. i'm not too worried about it though

Reply:
Could be that the testnet blockchain has been reset since the transaction creating the inputs to this recovery was originally created. I don't think it matters as long as the hex is valid. We should definitely test some recoveries before release though.

Reply:
if testnet blockchains do indeed get reset, then that would explain it